lib/components/RunFrame/RunFrame.tsx

```diff
@@ -245,13 +247,18 @@ export const RunFrame = (props: RunFrameProps) => {
       const evalResult = await worker
         .executeWithFsMap({
           entrypoint: props.entrypoint,
-          fsMap: fsMapObj,
+          fsMap: {
+            ...fsMapObj,
+            "manual-edits.json": JSON.stringify({
+              __esModule: true,
+              default: JSON.parse(fsMapObj["manual-edits.json"]),
+            }),
```

Comment:
You shouldnt need to do this transform here, its an eval bug if you do

Reply:
Hmm.. I did add this in the `eval` because I was facing this issue in eval as well 🤔